source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRichTextBlockRenderer.mm

```diff
@@ -6,45 +6,44 @@
 //
 
 #import "ACRRichTextBlockRenderer.h"
-#import "ACRContentHoldingUIView.h"
-#import "RichTextBlock.h"
-#import "TextRun.h"
-#import "ACRAggregateTarget.h"
-#import "HostConfig.h"
-#import "MarkDownParser.h"
-#import "ACRView.h"
-#import "ACOHostConfigPrivate.h"
-#import "ACOBaseCardElementPrivate.h"
 #import "ACOBaseActionElementPrivate.h"
+#import "ACOBaseCardElementPrivate.h"
+#import "ACOHostConfigPrivate.h"
+#import "ACRAggregateTarget.h"
+#import "ACRContentHoldingUIView.h"
+#import "ACRLongPressGestureRecognizerFactory.h"
 #import "ACRUILabel.h"
+#import "ACRView.h"
 #import "DateTimePreparsedToken.h"
 #import "DateTimePreparser.h"
+#import "HostConfig.h"
+#import "MarkDownParser.h"
+#import "RichTextBlock.h"
+#import "TextRun.h"
 #import "UtiliOS.h"
-#import "ACRLongPressGestureRecognizerFactory.h"
 
 @implementation ACRRichTextBlockRenderer
 
-+ (ACRRichTextBlockRenderer *)getInstance
-{
++ (ACRRichTextBlockRenderer *)getInstance {
     static ACRRichTextBlockRenderer *singletonInstance = [[self alloc] init];
     return singletonInstance;
 }
 
-+ (ACRCardElementType)elemType
-{
++ (ACRCardElementType)elemType {
     return ACRRichTextBlock;
 }
 
 - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
-          rootView:(ACRView *)rootView
-            inputs:(NSMutableArray *)inputs
-   baseCardElement:(ACOBaseCardElement *)acoElem
-        hostConfig:(ACOHostConfig *)acoConfig;
+           rootView:(ACRView *)rootView
```

Comment:
nit: extra spaces here make the ':' not to be aligned

Reply:
@nesalang, hopefully we can get the clang-formatting solution in for iOS as well? What was teh plan of action here for stylistic policies/applcations to iOS again? Are we looking at clang? 

---
In reply to: [322855682](https://github.com/microsoft/AdaptiveCards/pull/3430#discussion_r322855682) [](ancestors = 322855682)

Reply:
@shalinijoshi19 clang format is essentially ready. i closed the previous clang format pr because review wasn't completed fast enough to avoid merge conflicts with changes that are merged. So i will reopen the branch with clang format changes today.
the plan for stylistic polices is we are following apple's ios objective-c coding guide line. 

---
In reply to: [322900079](https://github.com/microsoft/AdaptiveCards/pull/3430#discussion_r322900079) [](ancestors = 322900079,322855682)

Reply:
@almedina-ms, this is result of running clang format which follows apple's coding style. so it would keep it as it.

---
In reply to: [322855682](https://github.com/microsoft/AdaptiveCards/pull/3430#discussion_r322855682) [](ancestors = 322855682)